src/collector/alloc.rs

```diff
@@ -19,6 +19,7 @@ pub enum DeallocationAction {
     DoNothing,
     RunDrop,
     RunFinalizer { finalize_ptr: *const dyn Finalize },
+    BoxDrop,
```

Comment:
Alphabetize!

Reply:
This has been updated in the most recent commit